hasura.planx.uk/metadata/tables.yaml

```diff
@@ -2468,7 +2477,7 @@
     name: temp_data_migrations_audit
     schema: public
   object_relationships:
-    - name: flows
+    - name: flow
```

Comment:
This is an `object_relationship` and should therefore be a singular rather than plural name, my mistake yesterday !